Update react-without-jsx.md (#92)
reactjs/hu.reactjs.org: content/docs/react-without-jsx.md

---
id: react-without-jsx
title: React JSX nélkül
permalink: docs/react-without-jsx.html
---

> Nem kötelező a JSXet választani, ha Reactot használsz. A React JSX nélkül különösen hasznos, ha nem akarsz külön fordítási lépést létrehoznod a build környezetedben.

Reviewer: ```suggestion
Nem kötelező a JSX-et választani, ha Reactet használsz. A React JSX nélkül különösen hasznos, ha nem akarsz külön fordítási lépést létrehozni a build környezetedben.
```

> Minden egyes JSX elem csak egy rövidítés a `React.createElement(component, props, ...children)` metódus hívására. Így, mindent amit JSXel le tudsz írni, meg tudod csinálni JSX nélkül, síma JavaScriptet használva is.

Reviewer: ```suggestion
Minden egyes JSX elem csak egy rövidítés a `React.createElement(component, props, ...children)` metódus hívására. Így, mindent amit JSX-szel le tudsz írni, meg tudod csinálni JSX nélkül is, sima JavaScriptet használva.
```

> Például, ez a JSXet használó kódrészlet

Reviewer: ```suggestion
Például, ez a JSX-szet használó kódrészlet:
```

);
```

> átírható a következő, JSXet nem használó kódra:

Reviewer: ```suggestion
átírható a következő, JSX-szet nem használó kódra:
```

  document.getElementById('root')
);
```

> Próbáld ki [az online Babel fordítót](babel://jsx-simple-example), ha szeretnél több példát látni rá, hogyan alakítjuk át a JSXet JavaScriptté.

Reviewer: It looks like there is a line misalignment from here downwards. Please fix that, so it looks exactly like the original (English) text.

Author: Sure!

A komponens megadható sztringként, a `React.Component` alosztályaként vagy egy sima függvényként.

> Ha úgy érzed, hogy túl időigényes mindig kigépelni a `React.createElement`et, egy gyakori megoldás, a függvény egy rövidítéshez rendelése:

Reviewer: ```suggestion
Ha úgy érzed, hogy túl időigényes mindig kigépelni a `React.createElement`-et, egy gyakori megoldás, a függvény egy rövidítéshez rendelése:
```

  document.getElementById('root')
);
```

> A `React.createElement`nek ezt a rövidített formáját használva, már sokkal kényelmesebben React kódot írni, még JSX nélkül is.

Reviewer: ```suggestion
A `React.createElement`-nek ezt a rövidített formáját használva, már sokkal kényelmesebb React kódot írni, még JSX nélkül is.
```
